src/Suave/Suave.fsproj

```diff
@@ -196,4 +196,89 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
```

Comment:
Yes exactly. No problem :). I've done something similar in Logary; it's really about taking a JSON and creating a DTO.

You're going good job so far, btw. =)